common/core/desktop/include/keyman/keyboardprocessor.h.in

```diff
@@ -494,6 +494,7 @@ enum km_kbp_action_type {
           // for applications where context is cached, this clears the context;
           // for applications where context is read from the focused text store,
           // the context is just re-read and markers flushed.
+  KM_KBP_IT_CAPSLOCK    = 8,  // Enable or disable capsLock
```

Comment:
:+1:

Reply:
Should add this flag to the [15.0 wiki](https://github.com/keymanapp/keyman/wiki/Keyman-15.0-New-Functionality) too

Reply:
Added to release notes section of A15S9 sprint retrospective.